KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaLiveManager.java

```diff
@@ -127,7 +128,7 @@ public boolean removeReferrer(ILiveEntryReferrer obj) {
 		}
 
 		public LiveEntryCache(KalturaLiveEntry liveEntry, KalturaMediaServerIndex serverIndex, String applicationName) {
-			this(liveEntry);
+			// Check if this constructor is even relevant? Might be possible to delete?
```

Comment:
u can use an a set of tools called compiler & linker to find out if we use this CTOR. comment it out and try to build :)